src/Umbraco.Cms.Integrations.Script.Cookiebot/Umbraco.Cms.Integrations.Script.Cookiebot.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Umbraco.Cms.Web.Website" version="[10.3.0,12)" />
+		<PackageReference Include="Umbraco.Cms.Web.BackOffice" version="[10.3.0,12)" />
+	</ItemGroup>
+
+	<ItemGroup>
```

Comment:
You could have a look at using a Razor class library here (like we do in Forms and Deploy now for static assets).  It's has a couple of small advantages, in that it simplifies/removes the needs for some of this "targets" configuration, and it means the view file is never on disk and looking like it might be editable.

See docs [here](https://learn.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-7.0&tabs=visual-studio).